schema/constraint_any_expression.go

```diff
@@ -0,0 +1,30 @@
+package schema
+
+import (
+	"github.com/zclconf/go-cty/cty"
+)
+
+// AnyExpression TODO
+type AnyExpression struct {
```

Comment:
Can you also add a basic compilation test to `schema/constraint_test.go` for this new constraint?

e.g.
```go
_ Constraint = AnyExpression{}
```
just to make sure it complies with the `Constraint` interfaces and continues to be compliant.